site/hugo.yaml

```diff
@@ -236,6 +236,9 @@ markup:
     noHl: false
     style: 'monokai'
     tabWidth: 4
+  goldmark:
+    renderer:
+      unsafe: true
```

Comment:
I'm fine with this change. But I'd still prefer to avoid using html tag if possible in markdown.